src/pybind/mgr/balancer/module.py

```diff
@@ -292,7 +292,7 @@ class Module(MgrModule):
         {
             'name': 'upmap_max_iterations',
             'type': 'uint',
-            'default': 10,
+            'default': 60,
```

Comment:
is this change part of this PR? if yes, why?

Reply:
I made a mistake this isn't right